roles/katello_test/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+foreman_development_dir: /home/vagrant/foreman/
+test_user: vagrant
+test_vcr_mode: all
```

Comment:
Are any of these intended to be configurable by someone running this role based n their environment? If yes, I would move these to `defaults/main.yml` and then scope them on the role by prefixing `katello_test` on each.